pkg/runctl/metrics/pipelineruns_periodic_test.go

```diff
@@ -163,11 +160,11 @@ func Test_pipelineRunsPeriodic_NonNewRun(t *testing.T) {
 
 				run := &stewardapi.PipelineRun{}
 				run.Status.State = state
+				run.Status.StateDetails.State = state
```

Comment:
Why is this required? 
In [pkg/runctl/metrics/pipelineruns_periodic.go](https://github.com/SAP/stewardci-core/pull/322/files?diff=split&w=0#diff-10097660fa5d0bf4a5c58c34f41882e61de0888c277e5ab8e53fe3309e89b3da) line 76 run.Status.State is used. We should decide to use one of them run.Status.State or run.Status.StateDetails.State

Reply:
This information is redundantly contained in a PipelineRunStatus. The metrics implementation may use the one or the other field. Therefore, the test sets both (as in real objects).

Reply:
However, it was inconsistent in the two tests. I've added this line in `Test_..._NewRun`, too.